cronjobs/FileLoader.py

```diff
@@ -0,0 +1,128 @@
+import os, sys
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+
+from models import ScheduleHistory, CallingPointHistory
+from darwinpush.messagefactories.xml import ScheduleXMLMessageFactory
+import darwinpush.xb.pushport as pp
+
+import re
+
+class ScheduleFileLoader:
+
+    def __init__(self, file_location):
+        assert(file_location)
+        self.file_location = file_location
+
+    def update_daily_schedules(self):
+
+        collect_mode=False
+        journey_buffer = []
+
+        for line in open(self.file_location):
+
+            if '<Journey' in line:
+                assert(not collect_mode)
+                collect_mode=True
+
+            elif '</Journey' in line:
+                assert(collect_mode)
+
+                journey_buffer.append(line)
+                journey_string = "".join(journey_buffer)
```

Comment:
This might cause bugs for input files that look like:

    ... </journey><journey> ...
    ...
    </journey><journey><something...>

It's not this line just the way the parser is built here.

Solution would be to care about the position of `<Journey` and `</Journey` in the current line when making the buffer, to ignore anything before the open tag or after the close tag.